03-Bash/kernel_git/kernel_hm.sh

```diff
@@ -16,17 +16,28 @@ if [ ! -d "$1" ]; then
 fi
 
 cd $1
+
+# Print the quantity of files with extention .c, .cpp and *.py.
 echo Quantity of files with extention .c in target diretory:
 ls -lR | grep --count \.c$
 echo Quantity of files with extention .cpp in target diretory:
 ls -lR | grep --count \.cpp$
 echo Quantity of files with extention .py in target diretory:
 ls -lR | grep --count \.py$
 
+# Find the quantity of commits with word 'revert' in commit name.
 echo
 echo Quantity of commits with word \'revert\'
 git rev-list HEAD --count --grep='revert'
 
+# How many lines for each author are in *.py files.
+for pfile in $(find -name "*.py")
+do
+  echo "file: $pfile"
+  git blame --line-porcelain ${pfile} | sed -n 's/^author //p' | sort | uniq -c | sort -rn
```

Comment:
This command add statistic for each file separately but the statistic for all files together is requested.